Cargo.toml

```diff
@@ -16,7 +16,7 @@ regex = "1.8.2"
 termcolor = "1.2.0"
 tree-sitter = "0.20.10"
 tree-sitter-javascript = "0.20.0"
-tree-sitter-rust = "0.20.3"
+tree-sitter-rust = { git = "https://github.com/helixbass/tree-sitter-rust", rev = "6327788" }
```

Comment:
The approach was to fork the tree-sitter grammar and tell it to attempt to parse `(...)`/`[...]`-style macro invocations as a sequence of expressions

I opened a [dummy PR](https://github.com/helixbass/tree-sitter-rust/pull/1)